harmony/config.py

```diff
@@ -55,12 +56,20 @@ def harmony_hostname(self):
         """Returns the hostname for this Config object's Environment."""
         return HOSTNAMES[self.environment]
 
+    @property
+    def url_scheme(self) -> str:
+        return 'http' if self.environment == Environment.LOCAL else 'https'
```

Comment:
Sandbox using an internal load balancer only has an http listener (so sandbox and local should both be http).